### ISSUE TITLE
Unable to use Azure blob for remote

### ISSUE THREAD

**Issue by saurav2608** [iterative/dvc#2953]
**Please provide information about your setup**
DVC version-0.76.0, 
Platform and method of installation-pip on Mac(10.14.6)

I execute `dvc remote add -f myremote azure://abc/dvc` (abc exists on my Azure account)

I also add con string by `dvc remote modify myremote connection_string "DefaultEndpointsProtocol=https;AccountName=abc;AccountKey=***;EndpointSuffix=core.windows.net" --local`

**Error (without -v)**
Error on `dvc add data/data.xml`

Stage is cached, skipping.                                                                                              
  0% Add|                                                                              |0.00/1.00 [00:00<?,     ?file/s]ERROR: unexpected error - __init__() got an unexpected keyword argument 'verify'

Having any troubles? Hit us up at https://dvc.org/support, we are always happy to help!
  0% Add|                                                                              |0.00/0.00 [00:00<?,     ?file/s]

**Error (with -v)**
```

DEBUG: Trying to spawn '['/Users/saurav/yes/envs/code/bin/python', '/Users/saurav/yes/envs/code/bin/dvc', 'daemon', '-q', 'updater']'
DEBUG: Spawned '['/Users/saurav/yes/envs/code/bin/python', '/Users/saurav/yes/envs/code/bin/dvc', 'daemon', '-q', 'updater']'
DEBUG: PRAGMA user_version;                                             
DEBUG: fetched: [(3,)]
DEBUG: CREATE TABLE IF NOT EXISTS state (inode INTEGER PRIMARY KEY, mtime TEXT NOT NULL, size TEXT NOT NULL, md5 TEXT NOT NULL, timestamp TEXT NOT NULL)
DEBUG: CREATE TABLE IF NOT EXISTS state_info (count INTEGER)
DEBUG: CREATE TABLE IF NOT EXISTS link_state (path TEXT PRIMARY KEY, inode INTEGER NOT NULL, mtime TEXT NOT NULL)
DEBUG: INSERT OR IGNORE INTO state_info (count) SELECT 0 WHERE NOT EXISTS (SELECT * FROM state_info)
DEBUG: PRAGMA user_version = 3;
DEBUG: Path .dvc/cache/a3/04afb96060aad90176268345e10355 inode 29855815                                                 
DEBUG: SELECT mtime, size, md5, timestamp from state WHERE inode=?                                                      
DEBUG: fetched: [('1576305886325939456', '37916850', 'a304afb96060aad90176268345e10355', '1576307774222853120')]        
DEBUG: UPDATE state SET timestamp = ? WHERE inode = ?                                                                   
DEBUG: cache '.dvc/cache/a3/04afb96060aad90176268345e10355' expected 'a304afb96060aad90176268345e10355' actual 'a304afb96060aad90176268345e10355'
DEBUG: Path data/data.xml inode 29857786                                                                                
DEBUG: SELECT mtime, size, md5, timestamp from state WHERE inode=?                                                      
DEBUG: fetched: [('1576305886325939456', '37916850', 'a304afb96060aad90176268345e10355', '1576307774222640128')]        
DEBUG: UPDATE state SET timestamp = ? WHERE inode = ?                                                                   
DEBUG: Saving 'data/data.xml' to '.dvc/cache/a3/04afb96060aad90176268345e10355'.                                        
DEBUG: Path .dvc/cache/a3/04afb96060aad90176268345e10355 inode 29855815                                                 
DEBUG: SELECT mtime, size, md5, timestamp from state WHERE inode=?                                                      
DEBUG: fetched: [('1576305886325939456', '37916850', 'a304afb96060aad90176268345e10355', '1576308085010927104')]        
DEBUG: UPDATE state SET timestamp = ? WHERE inode = ?                                                                   
DEBUG: cache '.dvc/cache/a3/04afb96060aad90176268345e10355' expected 'a304afb96060aad90176268345e10355' actual 'a304afb96060aad90176268345e10355'
DEBUG: Created 'reflink': .dvc/cache/.cache_type_test_file -> data/.vozB3nxQK6CcTBQuaAYuaE                              
DEBUG: Removing 'data/.vozB3nxQK6CcTBQuaAYuaE'                                                                          
DEBUG: Removing '.dvc/cache/.cache_type_test_file'                                                                      
DEBUG: Removing 'data/data.xml'                                                                                         
DEBUG: Created 'reflink': .dvc/cache/a3/04afb96060aad90176268345e10355 -> data/data.xml                                 
DEBUG: Path data/data.xml inode 29858047                                                                                
DEBUG: REPLACE INTO link_state(path, inode, mtime) VALUES (?, ?, ?)                                                     
DEBUG: Path data/data.xml inode 29858047                                                                                
DEBUG: SELECT mtime, size, md5, timestamp from state WHERE inode=?                                                      
DEBUG: fetched: []                                                                                                      
DEBUG: INSERT INTO state(inode, mtime, size, md5, timestamp) VALUES (?, ?, ?, ?, ?)                                     
DEBUG: Path .dvc/cache/a3/04afb96060aad90176268345e10355 inode 29855815                                                 
DEBUG: SELECT mtime, size, md5, timestamp from state WHERE inode=?                                                      
DEBUG: fetched: [('1576305886325939456', '37916850', 'a304afb96060aad90176268345e10355', '1576308085015379968')]        
DEBUG: UPDATE state SET mtime = ?, size = ?, md5 = ?, timestamp = ? WHERE inode = ?                                     
Stage is cached, skipping.                                                                                              
  0% Add|                                                                              |0.00/1.00 [00:00<?,     ?file/s]ERROR: unexpected error - __init__() got an unexpected keyword argument 'verify'


Having any troubles? Hit us up at https://dvc.org/support, we are always happy to help!
  0% Add|                                                                              |0.00/0.00 [00:00<?,     ?file/s]
```

**Comment by saurav2608** [comment]
On further investigation, I found the issue to be somewhere else. I may create a different issue later. 
Broadly v77 of dvc solved the above. But I got a conflict with azure-storage-blob==2.1.
I think dvc needs 2.1 whereas by pipeline is written for azure-storage-blob==12+. 

As on now is am switching envs to solve my problem.

**Comment by efiop** [comment]
Hi @saurav2608 ! Indeed, that `init` issue was resolved in 0.77. If you are having issues with azure-storage-blob compatibility, consider installing dvc using things like pipenv. I see that you are on mac, so I would probably install dvc through brew with `brew install dvc`, which would also create a separate virtualenv for dvc and would provide a shim, so that it doesn't collide with other requirements for your project.

Thanks for the feedback! 🙂

**Comment by saurav2608** [comment]
Thanks for the prompt note. I intend to use DVC in a CI environment.  I will have to stick to using pip/conda.
